packages/serverless-api/typedoc.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "@twilio-labs/serverless-api",
-  "mode": "modules",
```

Comment:
To confirm, is. this because `mode` is removed from TypeDocs?

Reply:
Yep